src/olympia/blocklist/management/commands/validate_mlbf.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand
+
+import olympia.core.logger
+from olympia.blocklist.mlbf import MLBF
+
+
+log = olympia.core.logger.getLogger('z.amo.blocklist')
+
+
+class Command(BaseCommand):
+    help = 'Validates a given MLBF directory'
```

Comment:
nit: It'd be nice if the help text was a little bit descriptive

Reply:
Better?